.github/workflows/project-card-moved.yml

```diff
@@ -17,3 +17,14 @@ jobs:
         with:
           add-labels: "confirmed"
           repo-token: ${{ secrets.NEO4J_TEAM_GRAPHQL_PERSONAL_ACCESS_TOKEN }}
+  add_high_priority_label:
+    if: github.event.project_card.column_id == '17286751'
```

Comment:
This is going to be fun someday

Reply:
Hahaha, it's the only way to do it, crazy right?!